controllers/network/network_controller.go

```diff
@@ -127,6 +127,18 @@ func add(mgr manager.Manager, r *ReconcileNetwork) error {
 		return err
 	}
 
+	ordererPredictFuncs := predicate.Funcs{
+		UpdateFunc: r.OrdererUpdateFunc,
+	}
+
+	err = c.Watch(&source.Kind{Type: &current.IBPOrderer{}}, &handler.EnqueueRequestForOwner{
```

Comment:
Just monitor those orderers has OwnerReferences to network. These orderers are root orderer.